enhancements/insights/insights-operator-runtime.md

```diff
@@ -0,0 +1,532 @@
+in the Insights operator but that does not give a limit on the number of containers. It is also very likely that some of the pods are replicas and their containers would only count for 1 `WorkloadContainerShape` in their gathered data.
+
+#### UX Risk & Mitigation
+
+The additional runtime info added to the gathered data will be exposed to the user though the archives generated by the Insights operator. As the values of the field are hashed, the user will not be able to determine their corresponding content.
+
+Exposing these data to the user with a clean user interface (eg within console.redhat.com) is out of scope of this proposal.
+
+### Drawbacks
+
+If implemented, Red Hat will document the addition of this new collection. We  introduce the risk that some number of customers who would have or are sending telemetry data via the Insights operator will choose to turn it off because they are uncomfortable sharing this additional data. 
+
+The insights-operator-runtime needs to be continually updated to detect new workloads  that Red Hat wants information about. 
+This can be done iteratively and will require an updated reference to the insights-operator-runtime image in the Insights Operator to take into account the additional detections.
+
+As the insights-operator-runtime is sanitizing the data extracted from the containers, it is not able to retrieve "raw" data (such as processes' name and command line) which limits its ability to detect emerging trends. The insights-operator-runtime must provide fingerprints executables for workloads that Red Hat wants to monitor.
+
+## Design Details
+
+### Open Questions [optional]
+
+#### insights-operator-runtime Image Availability
+
+The Insights Operator must be able to deploy the insights-operator-runtime from an image that is available by default in the cluster image registry.
+
+
+## Test Plan
+
+### insights-operator-runtime Test Plan
+
+The insights-operator-runtime can be tested in isolation to verify its capability to scan containers and collect the expected runtime info.
+
+Testing the insights-operator-runtime can be done outside of the Insights Operator test suite.
+It requires integration tests with applications based on the detectable runtime stacks. A end-to-end (e2e) test suite can be developed to verify the capabilities of the insights-operator-runtime.
+
+### Insights Operator Test Plan
+
+The integration tests for the Insights Operator will verify that it is able to properly deploy the insights-operator-runtime as a `DaemonSet` and have some smoke tests to validate that the additional runtime info can effectively be collected.
+
+## Graduation Criteria
+
+TBD
+
+### Dev Preview -> Tech Preview
+
+This enhancement would be behind the featuregate capabilities of OpenShift
+
+* Add a non-public flag to enable it on a cluster to collect runtime information
+* End User documentation is not required
```

Comment:
I think it is, see your earlier:

> If implemented, Red Hat will document the addition of this new collection. 

[Current docs][1] say:

> Cluster workload information transformed into discreet Secure Hash Algorithm (SHA) values, which allows Red Hat to assess workloads for security and version vulnerabilities without disclosing sensitive details

and as you point out in the enhancement, you're shifting that use-case a bit here.

[1]: https://docs.openshift.com/container-platform/4.15/support/remote_health_monitoring/about-remote-health-monitoring.html#insights-operator-what-information-is-collected_about-remote-health-monitoring

Reply:
+1, I'll update the RFE as this end-user statement would have to be updated.